tools/owl-vision/src/utils.ts

```diff
@@ -0,0 +1,39 @@
+import * as vscode from 'vscode';
+
+let statusMessage: any = undefined;
```

Comment:
should probably not be any. Maybe `ReturnType<vscode.window["createStatusBarItem"]>`?